### PR TITLE
Added Stack using Queue program in Cpp

### PR DESCRIPTION
**Pull request by rahulk32**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added Stack using Queue program in Cpp

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Issue #1715 is solved here)

### CODE REVIEW

**Comment by rahulk32** [comment]
This is for DWOC. Thankyou!

File: C-Plus-Plus/stack/stack_using_queue.cpp [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    }
+};
+
+int main()
+{
+    Stack s;
+    s.push(10);
+    s.push(100);
+    s.push(200);
+    s.push(250);
```
Comment (HarshCasper) [inline C-Plus-Plus/stack/stack_using_queue.cpp:72]: Change static input to dynamic input and take it from the User.

File: C-Plus-Plus/stack/stack_using_queue.cpp [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    cout<<"current size of stack: "<<s.size()<<endl;
+    cout<<"top element of stack: "<<s.top()<<endl;
+    s.pop();
+    cout<<"top element of stack: "<<s.top()<<endl;
+    cout<<"current size of stack: "<<s.size()<<endl;
```
Comment (HarshCasper) [inline C-Plus-Plus/stack/stack_using_queue.cpp:77]: Make a Switch Statement to take the responses from the User

File: C-Plus-Plus/stack/stack_using_queue.cpp [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        //put all the remaining elements in q1 to q2
+        while(!q1.empty())
+        {
+            int x=q1.front();
+            q2.push(x);
+            q1.pop();
+        }
+        //swap the names of two queues
+        queue<int> temp=q1;
+        q1=q2;
```
Comment (HarshCasper) [inline C-Plus-Plus/stack/stack_using_queue.cpp:39]: Format your Code with a formatter.

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by HarshCasper** [comment]
Hi @rahulk32 

Are you still working on it?

**Comment by HarshCasper** [comment]
Closing this because of no updates.

**Pull request closed without merging**